ocfweb/reservations/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
```

Comment:
i would vote for calling this stuff `lab_reservations` instead of just `reservations` (and updating the routes accordingly)

Reply:
I've done this -- good call